File: 2-Decisao/5.rb
#!/usr/bin/env ruby
nota1 = gets.chomp.to_f
nota2 = gets.chomp.to_f
media = (nota1 + nota2)/2

if media == 10.0
    puts 'Aprovado com Distincao'
elsif (7...10.0).include? media
    puts 'Aprovado'
elsif media < 7.0
    puts 'Reprovado'
end

File: 2-Decisao/5.py
#!/usr/bin/env python
nota1 = float(input())
nota2 = float(input())
media = (nota1 + nota2)/2

if media == 10.0:
    print 'Aprovado com Distincao'
elif 7.0 <= media < 10.0:
    print 'Aprovado'
elif media < 7.0:
    print 'Reprovado'